tests/system/ControllerTest.php

```diff
@@ -41,7 +41,7 @@ final class ControllerTest extends CIUnitTestCase
     private App $config;
 
     /**
-     * @var Controller
+     * @var Controller|object|null
```

Comment:
This is only  `Controller` or null.

```suggestion
     * @var Controller|null
```

Reply:
updated https://github.com/codeigniter4/CodeIgniter4/pull/6942/commits/e9167e9cb47cacd8d6da454bbb4d1f2a02087b87